src/server/bitops_family.cc

```diff
@@ -1239,16 +1239,8 @@ void SetBit(CmdArgList args, ConnectionContext* cntx) {
 // This are the "callbacks" that we're using from above
 std::string GetString(const PrimeValue& pv, EngineShard* shard) {
   std::string res;
-  if (pv.IsExternal()) {
-    auto* tiered = shard->tiered_storage();
-    auto [offset, size] = pv.GetExternalSlice();
-    res.resize(size);
 
-    std::error_code ec = tiered->Read(offset, size, res.data());
-    CHECK(!ec) << "TBD: " << ec;
-  } else {
-    pv.GetString(&res);
-  }
+  pv.GetString(&res);
```

Comment:
seems that `std::string GetString(const PrimeValue& pv, EngineShard* shard)` can be removed